src/WrapperApp/components/Simulation/SimulationPanel.tsx

```diff
@@ -89,21 +86,11 @@ export default function SimulationPanel({
 				handleLoadResults={handleLoadResults}
 				handleShowInputFiles={handleShowInputFiles}
 			/>
-			{demoMode ? (
-				<DemoCardGrid
```

Comment:
do we need DemoCardGrid at all ? maybe it could be removed as well ?

Reply:
Is it seen on this screen ?

![image](https://github.com/yaptide/ui/assets/7374733/1770e991-42d0-45c6-9f82-0bc2647569b7)

Reply:
Removed, i missed it previously